apps/backend/src/libs/plugins/authorization-plugin/authorization-plugin.ts

```diff
@@ -0,0 +1,64 @@
+import fp from "fastify-plugin";
+
+import { HTTPHeader } from "~/libs/modules/http/http.js";
+import { BaseToken } from "~/libs/modules/token/base-token.module.js";
+import { ServerHooks } from "~/libs/plugins/libs/enums/enums.js";
+import { TokenPayload } from "~/libs/types/types.js";
+import { UserService } from "~/modules/users/user.service.js";
+
+import { AuthError, ErrorMessage } from "./libs/types/enums/enums.js";
```

Comment:
You don't need enums folder here. `import { AuthError } from ~/modules/auth/auth.js` and `import { ErrorMessage } from ~/libs/enums/enums.js`